cli/types/cypress.d.ts

```diff
@@ -3019,6 +3019,17 @@ declare namespace Cypress {
      * @see https://on.cypress.io/configuration#experimentalModifyObstructiveThirdPartyCode
      */
     experimentalModifyObstructiveThirdPartyCode: boolean
+    /**
+     * Disables setting document.domain to the document's super domain on injection.
```

Comment:
documents or applications super domain?

Reply:
I think in this case they are the same thing, but application might be easier to digest?

Reply:
renamed in 86bea96010  (meant to do in dccf967ed9)